assets/js/base/components/checkout/form-step/index.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import Label from '../../label';
```

Comment:
That's just a code style preference but I think we have been quite consistent on it: we try to import JS files before CSS. So in this case `import './style.scss';` would be below `import Label...`.